### ISSUE TITLE
Pity Rate finished % in the chat is off by 1

### ISSUE THREAD

**Issue by Rmcwell** [geel9/runelite-hunter-rumours#58]
I got a kebbit rumour on the 1st catch and the % in that chat is still 0... shouldn't it be 5% (6% depending on rounding) The screenshot below shows it all.

I also hit the pity rate for sunlight moths and the chat said 99% instead of 100

It seems like the calculation for the % in the chat should have 1 added to it... for example it is doing 0/18 as a percent right now for dark kebbits instead of 1/18... and it did 141/142 instead of 142/142 for sunlight moths

![image](https://github.com/geel9/runelite-hunter-rumours/assets/27262971/73a2df82-0ce9-480a-8d67-052731abd8ef)

**Comment by DapperMickie** [comment]
Yeah I think the timing is off, should be an easy fix (update caught creatures before sending the message).
